src/renderer/services/client.js

```diff
@@ -137,20 +137,23 @@ export default class ClientService {
    * @param {Object} [query]
    * @param {Number} [query.page=1]
    * @param {Number} [query.limit=51]
+   * @param {String} [query.orderBy='rank:asc']
    * @return {Object[]}
    */
-  async fetchDelegates ({ page, limit } = {}) {
+  async fetchDelegates (options = {}) {
```

Comment:
@dated , I'm curious, why do you prefer using `options` instead of destructuring the object here?

Reply:
Just for conformity in regards to the other methods, such as `fetchWalletTransactions`.